lib/ui/page/home/tab/chats/widget/search_user_tile.dart

```diff
@@ -73,10 +73,7 @@ class SearchUserTile extends StatelessWidget {
           subtitle: [
             const SizedBox(height: 5),
             Text(
-              '${'label_num'.l10n}${'colon_space'.l10n}${(contact?.user.value?.user.value.num.val ?? user?.user.value.num.val)?.replaceAllMapped(
-                RegExp(r'.{4}'),
-                (match) => '${match.group(0)} ',
-              )}',
+              '${'label_num'.l10n}${'colon_space'.l10n}${(contact?.user.value?.user.value.num.toString() ?? user?.user.value.num.toString())}',
```

Comment:
Если комментарий выше верен, то тут тоже можно сократить чуть-чуть:

```suggestion
              '${'label_num'.l10n}${'colon_space'.l10n}${(contact?.user.value ?? user)?.user.value.num',
```

?